Corona-Warn-App/src/main/java/de/rki/coronawarnapp/statistics/StatsItem.kt

```diff
@@ -1,30 +1,117 @@
 package de.rki.coronawarnapp.statistics
 
-import de.rki.coronawarnapp.server.protocols.internal.stats.KeyFigureCardOuterClass
+import de.rki.coronawarnapp.server.protocols.internal.stats.KeyFigureCardOuterClass.KeyFigure
 import org.joda.time.Instant
+import timber.log.Timber
 
 data class StatisticsData(
-    val items: List<StatsItem>
+    val items: List<StatsItem> = emptyList()
 ) {
     val isDataAvailable: Boolean = items.isNotEmpty()
+
+    override fun toString(): String {
+        return "StatisticsData(cards=${items.map { it.cardType.name + " " + it.updatedAt }})"
+    }
 }
 
-sealed class StatsItem(val cardId: Int) {
+sealed class StatsItem(val cardType: Type) {
     abstract val updatedAt: Instant
-    abstract val keyFigures: List<KeyFigureCardOuterClass.KeyFigure>
+    abstract val keyFigures: List<KeyFigure>
+
+    enum class Type(val id: Int) {
+        INFECTION(1),
```

Comment:
minor: might be a little odd to start with 1 instead of 0

Reply:
Or is this referring to the `cardIdSequence` with values `[1,2,3,4]` of the tech spec?

Reply:
Yes. These are the `cardId`'s defined by the tech spec.